guides/common/modules/proc_migrating-a-host-to-use-pull-mode.adoc

```diff
@@ -0,0 +1,40 @@
+[id="migrating-a-host-to-use-pull-mode_{context}"]
+= Migrating a Host to Use Pull Mode
+
+To migrate a host from `katello-agent` to pull-based transport, you can use the `katello-pull-transport-migrate` utility.
+
+.Prerequisites
+* The host runs an Enterprise Linux derivative operating system.
+ifndef::satellite[]
+The utility should work on any system that uses Red Hat Subscription Manager, but it was tested only on Enterprise Linux.
+endif::[]
+* The host is registered to Katello.
```

Comment:
```suggestion
* The host is registered to {Project}.
```

Reply:
I'm not sure if it's a good idea to replace this, because the utility mentions in prerequisites specifically:
> Registered to Katello (script will exit if client is registered directly to RHSM)

Reply:
@adamruzicka Can you weigh in, please?

Reply:
This is still open IMHO.

Reply:
I tried to clarify that with the additional sentence. Does it help?

Reply:
Oh, sorry, I forgot to mention...
I discussed it with @adamruzicka on IRC and we came to this conclusion: The situation that the host is not registered via Katello is possible, so saying Katello instead of {Project} is more precise and probably appropriate in this case.